apps/emqx_auto_subscribe/src/emqx_auto_subscribe.erl

```diff
@@ -80,8 +80,15 @@ format(Rule = #{topic := Topic}) when is_map(Rule) ->
     }.
 
 update_(Topics) when length(Topics) =< ?MAX_AUTO_SUBSCRIBE ->
-    {ok, _} = emqx:update_config([auto_subscribe, topics], Topics),
-    update_hook();
+    case emqx_conf:update([auto_subscribe, topics],
+                          Topics,
+                          #{rawconf_with_defaults => true, override_to => cluster}) of
+        {ok, #{config := NewTopics}} ->
```

Comment:
using raw_config?

Reply:
Example: 15s, config will return 15000, raw_config will return 15s, we should return 15s to caller.

Reply:
will be same result by json:encode